smallfile_plugin.py

```diff
@@ -12,34 +12,47 @@
 import csv
 from arcaflow_plugin_sdk import plugin
 
-from smallfile_schema import WorkloadParams, WorkloadResults, WorkloadError, smallfile_schema, output_params_schema, output_results_schema, output_rsptimes_schema
+from smallfile_schema import (
+    WorkloadParams,
+    WorkloadResults,
+    WorkloadError,
+    smallfile_schema,
+    output_params_schema,
+    output_results_schema,
+    output_rsptimes_schema,
+)
 
 
-# The following is a decorator (starting with @). We add this in front of our function to define the metadata for our step.
+# The following is a decorator (starting with @). We add this in front of our
+# function to define the metadata for our step.
```

Comment:
Perhaps change the comment to become pydoc?

Reply:
Should I delete it then?